ngraph/core/include/ngraph/op/roi_pooling.hpp

```diff
@@ -32,7 +32,7 @@ namespace ngraph
                 ROIPooling() = default;
                 /// \brief Constructs a ROIPooling operation
                 ///
```

Comment:
Can we introduce `ROIPoolingTypes` and create a constructor with this enum?

Reply:
As agreed in internal discussion, supporting strong type for ROIPooling method will be addressed in a different ticket: 43566